src/tlog/keys.ts

```diff
@@ -14,18 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 import { createPublicKey, KeyObject } from 'crypto';
-import fs from 'fs';
-import path from 'path';
 import { crypto } from '../util';
 
 // Returns the set of trusted log keys which can be used to verify the
 // Signed Entry Timestamps in the log.
 export function getKeys(): Record<string, KeyObject> {
   // TODO: This should be be loaded via TUF
-  const pem = fs.readFileSync(
-    path.resolve(__dirname, '../../store/rekor.pub'),
-    'utf-8'
-  );
+  const pem = `
+-----BEGIN PUBLIC KEY-----
+MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE2G2Y+2tabdTV5BcGiBIx0a9fAFwr
+kBbmLSGtks4L3qX6yYY0zufBnhC8Ur/iy55GhWP/9A/bY2LhC30M9+RYtw==
+-----END PUBLIC KEY-----`
```

Comment:
This file should in theory be relocated by `ncc` > [webpack-asset-relocator-loader](https://github.com/vercel/webpack-asset-relocator-loader) but I had a go at this and looks like it doesn't like the typescript compiled code here as it's trying to pattern match on `path.*` and TS generates this: `path_1.default.resolve`.

What does seem to work is changing this to use `require.resolve`:

```suggestion
  const pem = fs.readFileSync(
    require.resolve('../../store/rekor.pub'),
    'utf-8'
  );
```